pkg/controller/signup.go

```diff
@@ -1,19 +1,20 @@
 package controller
 
 import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
 	"net/http"
 
-	errors2 "k8s.io/apimachinery/pkg/api/errors"
-
-	"github.com/codeready-toolchain/registration-service/pkg/verification"
-
 	"github.com/codeready-toolchain/registration-service/pkg/configuration"
 	"github.com/codeready-toolchain/registration-service/pkg/context"
 	"github.com/codeready-toolchain/registration-service/pkg/errors"
 	"github.com/codeready-toolchain/registration-service/pkg/log"
 	"github.com/codeready-toolchain/registration-service/pkg/signup"
+	"github.com/codeready-toolchain/registration-service/pkg/verification"
 
 	"github.com/gin-gonic/gin"
+	errors2 "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
`apierrors` may be a better alias than `errors2`

Reply:
fixed as part of: 60ca7377aabe427ed8f8e3d642380b1b8396454e